src/commands/registries/azure/DockerSiteCreateStep.ts

```diff
@@ -4,37 +4,78 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { WebSiteManagementClient, WebSiteManagementModels } from '@azure/arm-appservice'; // These are only dev-time imports so don't need to be lazy
+import { Site } from '@azure/arm-appservice/esm/models';

```

Comment:
Should this be a lazy import?

Reply:
It used for type definition in line 35. So it doesn't have to be lazy import. Added comment.

Reply:
cool